.github/workflows/playwright.yml

```
@@ -0,0 +1,27 @@
+name: Playwright Tests
+on:
```

yamllint (1.35.1)

[warning] truthy value should be one of [false, true]

(truthy)